core/snmp/api/src/main/java/org/opennms/netmgt/snmp/AbstractSnmpValue.java

```diff
@@ -174,4 +186,31 @@ else if (b >= -128 && b < -96) {
         }
         return true;
     }
+
+    static Set<Character> getAdditionalPrintableCharacters() {
+        if (ADDITIONAL_PRINTABLE_CHARACTERS == null) {
+            final String additionalCharactersString = System.getProperty(ADDITIONAL_PRINTABLE_CHARACTERS_PROPERTY, "");
+            if (Strings.isNullOrEmpty(additionalCharactersString)) {
```

Comment:
Please try to avoid pulling in Guava for API modules, it makes things a lot harder in Karaf, dependency-wise.

There are more useful bits in Guava, but if you're only including it for this one line, just do the slightly less pretty version. :)

```suggestion
            if (additionalCharactersString == null || additionalCharactersString.isEmpty()) {
```